webapp/channels/src/components/search_results/files_filter_menu.tsx

```diff
@@ -60,49 +60,89 @@ export default function FilesFilterMenu(props: Props): JSX.Element {
                 >
                     <Menu.ItemAction
                         ariaLabel={'All file types'}
-                        text={'All file types'}
+                        text={
```

Comment:
this actually expects a string because if you observe the code where the text goes it already has the span wrapped around it, <FormattedMessage component introduces another span when rendering. You can use useIntl hook for `formatMessage`